common/rpc/interceptor/namespace_test.go

```diff
@@ -58,11 +58,11 @@ var (
 	matchingAPIExcluded = map[string]struct{}{
 		"ListTaskQueuePartitions": {},
 		// Nexus APIs operate on a cluster scope, not a namespace scope.
-		"GetNexusService":            {},
-		"CreateNexusIncomingService": {},
-		"UpdateNexusIncomingService": {},
-		"ListNexusIncomingServices":  {},
-		"DeleteNexusIncomingService": {},
+		"GetNexusService":     {},
```

Comment:
Should be `GetNexusEndpoint`?

Reply:
There's no such method, it was removed in one of the previous refactors. Good catch.